step.yml

```diff
@@ -0,0 +1,54 @@
+
+title: |
+  Create ZIP
+summary: |
+  Create ZIP
+description: |
+  Create ZIP
+website: https://github.com/bitrise-steplib/steps-create-zip
+source_code_url: https://github.com/bitrise-steplib/steps-create-zip
+support_url: https://github.com/bitrise-steplib/steps-create-zip
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+type_tags:
+  - utility
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+
+toolkit:
+  go:
+    package_name: github.com/birmachera/bitrise-step-create-zip
+
+
+inputs:
+  - source_path: ./
+    opts:
+      title: "Source directory path"
+      summary: The path of the directory which you want to compress with zip.
+      is_expand: true
+      is_required: true
+      value_options: []
+
+  - destination: $source_path
```

Comment:
same as above :)

Reply:
It's optional and it's default value is the $source_path